src/pointer-analysis/dynamic_object_name.cpp

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************\
+
+Module: Dynamic object name
+
+Author: Marius-Constantin Melemciuc
+
+Date: April 2017
+
+@ Copyright Diffblue, Ltd.
+
+\*******************************************************************/
+
+#include "dynamic_object_name.h"
+
+/*******************************************************************\
+
+Function: get_dynamic_object_prefix
+
+  Inputs:
+
+ Outputs: The prefix of the dynamic-object,
+          composed of "value_set::dynamic_object".
+
+ Purpose: To generate a prefix for dynamic-objects suitable for use
+          in the LHS of value-set maps.
+
+\*******************************************************************/
+
```

Comment:
Personally I **do** agree with that, i. e. no blank line between the description and the implementation, but I found many places where it is present, for example, in `pointer-analysis/value_set.cpp`, lines 47, 74, 104, 141, 253. I added it to be consistent with these examples. 
Knowing this, please tell me what is your final decision. Thank you.